homeassistant/components/elmax/common.py

```diff
@@ -114,21 +126,21 @@ async def _async_update_data(self):
             _LOGGER.error("Refused username/password")
```

Comment:
Please include the message as the exception argument instead.

Reply:
Done.